v2/features/stats/jar/MultiLineChart.tsx

```diff
@@ -0,0 +1,109 @@
+import { useTranslation } from "next-i18next";
+import React, { FC } from "react";
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  Label,
+} from "recharts";
+import { AssetCoreData, JarChartData } from "./types";
+
+const aprAndApyData = (sortedAssetData: AssetCoreData) => {
+  return {
+    timestamp: sortedAssetData.timestamp,
+    jarApr: sortedAssetData.jarApr,
+    minApy: sortedAssetData.jarApr + sortedAssetData.farmMinApy,
+    maxApy: sortedAssetData.jarApr + sortedAssetData.farmMaxApy,
+  };
+};
+
+const Chart: FC<{ data: JarChartData; timeUnit: string }> = ({
+  data,
+  timeUnit,
+}) => {
+  const { t } = useTranslation("common");
+  
+  const assetData = data && data.assetData ? data.assetData[timeUnit] : [];
+  const sortedData = assetData
+    ? assetData.sort((a, b) => (a.timestamp > b.timestamp ? 1 : -1))
+    : [];
+  const chartData = sortedData.map(aprAndApyData);
+  return (
+    <LineChart
+      height={500}
+      width={1200}
```

Comment:
The fixed width will not be a viable strategy

Reply:
fixed